project/profiles/capacity4more/modules/c4m/content/c4m_content/c4m_content.module

```diff
@@ -765,18 +791,23 @@ function c4m_content_taxonomy_build_tree($items, $link_term_to_facet = array())
       $local_tree[] = $link;
     }
 
-    // If it's the last item on the local tree, it's the parent. We need
+    // If it's the first item on the local tree, it's the parent. We need
     // to distinguish between the parent and its children because the
     // parent needs to get a different style.
-    $root = array_pop($local_tree);
+    $root = array_shift($local_tree);
+    // Assign a dedicated class for trees that needs to be shifted right.
+    $class = (array_search($tree_index, $shift_right_array) !== FALSE) ? 'shift-right' : '';
```

Comment:
Better to use `$shift_right = array_search($tree_index, $shift_right_array) !== FALSE;`

and then to check `if $shift_right` then add the class